dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -875,7 +928,7 @@ boolean handleOutliers(ThroughputDistribution distribution, double throughput) {
             }
 
             // lean toward shrinking if cpuUtil is high
-            if (shrinkScore < 0.5) {
+            if ((shrinkScore < 0.5) && (poolSize > hangBufferPoolSize) && (priorKey > coreThreads)) {
```

Comment:
priorKey > coreThreads was not in all the branches in your previous version.  Is it appropriate to be used for all of them now?

Reply:
Yes, I think so ... :)
This is a change in this respin to reduce downward pressure on the pool size, because test data showed the pool being repeated forced down to coreThreads by coin tosses. This change (priorKey > coreThreads) avoids the coin-toss paths if the poolSize is already less than compareRange increments beyond coreThreads. 
The pool can still be pushed down to that range if the historical results show better throughput at those poolSizes, but will not be moved that way by the coin tosses.